sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -83,7 +80,7 @@ export type LogsColumnType = string;
 // @public
 export class LogsQueryClient {
     constructor(tokenCredential: TokenCredential, options?: LogsQueryClientOptions);
-    queryLogs(workspaceId: string, query: string, timespan: string, options?: QueryLogsOptions): Promise<QueryLogsResult>;
+    queryLogs(workspaceId: string, query: string, timespan: string, additionalWorkspaces?: string[], options?: QueryLogsOptions): Promise<QueryLogsResult>;
```

Comment:
I think `additionalWorkspaces` should be in the `options` bag - at least that's how I'm used to seeing additional optional params